src/index.js

```diff
@@ -4,13 +4,21 @@ import { BrowserRouter } from 'react-router-dom';
 import './scss/index.scss';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
+import { persistor } from './store/store';
+import { store } from './store/store';
+import { PersistGate } from 'redux-persist/integration/react';
+import { Provider } from 'react-redux';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
 	<React.StrictMode>
-		<BrowserRouter>
-			<App />
-		</BrowserRouter>
+		<Provider store={store}>
+			<PersistGate loading={null} persistor={persistor}>
```

Comment:
loading={null} это тоже самое если его не передать
```suggestion
			<PersistGate persistor={persistor}>
```